Lib9c/Action/BattleArena4.cs

```diff
@@ -68,6 +68,11 @@ public class BattleArena4 : GameAction
 
         public override IAccountStateDelta Execute(IActionContext context)
         {
+            if (context.BlockIndex < 5048399)
```

Comment:
Can we change `5048399` to const?
Something like `BlockPolicySource.LastBlockIndexOfV100291`.